src/gluonts/shell/__init__.py

```diff
@@ -1,17 +0,0 @@
-# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-# permissions and limitations under the License.
-
-# Relative imports
-from ._base import PathsEnvironment
-
-__all__ = ['PathsEnvironment']
```

Comment:
I need to re-add the file.

But do we want to expose anything here?

Reply:
I don't think there is a need for that for now.